app/flows/register_a_birth_flow/outcomes/oru_result.erb

```diff
@@ -185,17 +185,17 @@ Depending on the document, you'll be asked to provide an original version or a p
   Print out and fill in the [application form](/government/publications/application-to-register-an-overseas-birth).
 
 
-  ##3. Pay
-
   <% if !calculator.in_the_uk? && calculator.registration_country == 'algeria' %>
+
+  ##3. Pay
```

Comment:
You have moved this heading which means it won't show up for anyone not choosing UK or Algeria. Was that intentional? It doesn't seem right.